docs/_running-tests/index.md

```diff
@@ -2,79 +2,77 @@
 layout: page
 title: Running Tests
```

Comment:
This isn't really about running tests anymore, but only about setting up to run the server.